src/main/java/pages/IndexPage.java

```diff
@@ -0,0 +1,159 @@
+package pages;
+
+import static org.testng.Assert.assertEquals;
+
+import java.util.List;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.testng.Assert;
+import pages.components.HeaderMenu;
+import pages.components.LeftSideMenu;
+
+public class IndexPage {
+
+    private HeaderMenu headerMenu;
+    private LeftSideMenu leftSideMenu;
+    private WebDriver driver;
+
+    public IndexPage(WebDriver driver) {
+        this.driver = driver;
+        PageFactory.initElements(driver, this);
+        headerMenu = new HeaderMenu(driver);
+        leftSideMenu = new LeftSideMenu(driver);
+    }
+
+    public HeaderMenu getHeaderMenu() {
+        return headerMenu;
+    }
+
+    public LeftSideMenu getLeftSideMenu() {
+        return leftSideMenu;
+    }
+
+    //Login section
+    @FindBy(css = "li.dropdown.uui-profile-menu")
+    private WebElement userIcon;
+
+    @FindBy(css = "[id = 'name']")
+    private WebElement userName;
+
+    @FindBy(css = "[id = 'password']")
+    private WebElement userPassword;
+
+    @FindBy(css = "[id = 'login-button']")
+    private WebElement loginButton;
+
+    @FindBy(css = "div.logout")
+    private WebElement logoutButton;
+
+    @FindBy (id = "user-name")
+    private WebElement fullUserName;
+
+    //Icon section
+    @FindBy (css = ".icons-benefit.icon-practise")
+    private WebElement iconPractice;
+
+    @FindBy (css = ".icons-benefit.icon-custom")
+    private WebElement iconCustom;
+
+    @FindBy (css = ".icons-benefit.icon-multi")
+    private WebElement iconMulti;
+
+    @FindBy (css = ".icons-benefit.icon-base")
+    private WebElement iconBase;
+
+    @FindBy (css = ".benefit-txt")
+    private List<WebElement> textsUnderIcons;
+
+    //iFrame
+    @FindBy (id = "frame")
+    private WebElement iframe;
+
+    @FindBy (css = "#frame-button")
+    private WebElement iframeButton;
+
+    public void openIndexPage() {
+        driver.navigate().to("https://jdi-testing.github.io/jdi-light/index.html");
```

Comment:
changed, URL is moved to properties file

Reply:
You load the same properties file twice
 - in AbstractSettingClass
 - in IndexPage
Please move properties initialization into a separate class. You can also try to implement Singleton pattern in that class since we need to load properties once.

Reply:
Singleton class is implemented